SPECS/quota/quota.spec

```diff
@@ -231,15 +206,6 @@ Linux/UNIX environment.
 %patch7 -p1
```

Comment:
Nit: could use autosetup since there are many addition/ deletion from patchlist

Reply:
Thanks for the suggestion! Fixed!

Reply:
Thanks for the suggestion! Fixed!